course3/homework-string/my_string.cc

```diff
@@ -0,0 +1,41 @@
+#include <iostream>
+#include <cstring>
+#include "my_string.h"
+
+MyString::MyString(){
+	buffer = std::unique_ptr<char[]>(new char[0]);
```

Comment:
Why allocate 0 bytes? This may have unexpected results. It would be much better to leave the pointer null